src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -20,4 +30,10 @@ public final class VisionConstants {
     public static final double spinD = 0.0002;
     public static final double spinSetpoint = 0.0;
     public static final double aprilTagArea = 28908;
+
+    public static final AprilTagFieldLayout kTagLayout = AprilTagFieldLayout.loadField(AprilTagFields.kDefaultField);
+    public static final Transform3d kRobotToCam = new Transform3d(new Translation3d(0.1, 0.0, 0.1),
```

Comment:
These translation and rotation values should be constants and the robot to cam transform should be constructed in vision.java.
You also need 2 sets for the 2 cameras